OrbitGl/ProcessesDataView.cpp

```diff
@@ -105,6 +106,13 @@ uint32_t ProcessesDataView::GetSelectedProcessId() const {
   return selected_process_id_;
 }
 
+uint32_t ProcessesDataView::GetFirstProcessId() const {
+  if (m_Indices.empty()) {
+    return 0;
```

Comment:
shouldn't you use either a constant or at least "-1"?

Reply:
We are dealing with `uint32_t`s and we are using 0 as invalid pid in so many places, I wouldn't change this.